src/client.js

```diff
@@ -487,6 +535,73 @@ class Admin {
     };
   }
 
+  v2() {
+    const api = this._v2;
+    const TODOnotImplemented = ()=>  { throw new Error('Not yet implemented'); };
+    return {
+      apps: (groupId)  => {
+        const groupUrl = `/groups/${groupId}/apps`;
+        return {
+          list: () => api._get(groupUrl),
+          create: (data, options) => {
+            let query = (options && options.defaults) ? '?defaults=true' : '';
+            return api._post(groupUrl + query, data);
+          },
+          app: (appId) => {
+            const appUrl = `${groupUrl}/${appId}`;
+            return {
+              get: () => api._get(appUrl),
+              remove: () => api._delete(appUrl),
+              pipelines: () => ({
+                list: () => api._get(`${appUrl}/pipelines`),
+                create: (data) => api._post( `${appUrl}/pipelines`, data),
+                pipeline: (pipelineId) => {
+                  const pipelineUrl = `${appUrl}/pipelines/${pipelineId}`;
+                  return {
+                    get: ()=> api._get(pipelineUrl),
+                    remove: ()=> api._delete(pipelineUrl),
+                    update: (data) => api._put(pipelineUrl, data)
+                  };
+                }
+              }),
+              values: () => ({
+                list: () => api._get(`${appUrl}/values`),
+                create: (data) => api._post( `${appUrl}/values`, data),
+                value: (valueId) => {
+                  const valueUrl = `${appUrl}/values/${valueId}`;
+                  return {
+                    get: ()=> api._get(valueUrl),
+                    remove: ()=> api._delete(valueUrl),
+                    update: (data) => api._put(valueUrl, data)
+                  };
+                }
+              }),
+              services: () => ({
+                list: () => api._get(`${appUrl}/services`),
+                create: (data) => api._post(`${appUrl}/services`, data),
+                service: (serviceId) => ({
+                  get: () => api._get(`${appUrl}/services/${serviceId}`),
+                  remove: () => api._delete(`${appUrl}/services/${serviceId}`),
+                  config: ()=> ({
+                    get: () => api._get(`${appUrl}/services/${serviceId}/config`),
+                    update: (data) => api._patch(`${appUrl}/services/${serviceId}/config`, data)
+                  })
+                })
+              }),
+              pushNotifications: TODOnotImplemented,
+              users: TODOnotImplemented,
+              sandbox: TODOnotImplemented,
```

Comment:
I think we are calling this dev in the future, right?

Reply:
yes, true. updated.